lib/std/math/fabs.zig

```diff
@@ -16,86 +15,28 @@ const maxInt = std.math.maxInt;
 ///  - fabs(nan)   = nan
 pub fn fabs(x: anytype) @TypeOf(x) {
     const T = @TypeOf(x);
-    return switch (T) {
-        f16 => fabs16(x),
-        f32 => fabs32(x),
-        f64 => fabs64(x),
-        f128 => fabs128(x),
-        else => @compileError("fabs not implemented for " ++ @typeName(T)),
-    };
-}
-
-fn fabs16(x: f16) f16 {
-    var u = @bitCast(u16, x);
-    u &= 0x7FFF;
-    return @bitCast(f16, u);
-}
-
-fn fabs32(x: f32) f32 {
-    var u = @bitCast(u32, x);
-    u &= 0x7FFFFFFF;
-    return @bitCast(f32, u);
-}
-
-fn fabs64(x: f64) f64 {
-    var u = @bitCast(u64, x);
-    u &= maxInt(u64) >> 1;
-    return @bitCast(f64, u);
-}
+    const IntT = std.meta.Int(.unsigned, @bitSizeOf(T));
```

Comment:
If its unsigned, it should be something along `UintT` or `UT` to have consistency with `u32,i32` etc.

Reply:
I used `IntT` to be consistent with the rest of the standard library, ex.

https://github.com/ziglang/zig/blob/d29871977f97b50fe5e3f16cd9c68ebeba02a562/lib/std/fmt/parse_hex_float.zig#L12-L15

I think specifying that it's for bit hacks is better though, as I don't need to use `maxInt` to get all ones, so I'll go with with `TBits` to mean a raw bit interpretation of the float, which is what it is.